### DISCUSSION TITLE
Documentation for `useActionGroup`

### DISCUSSION THREAD

**Discussion by ritz078** [adobe/react-spectrum#4427] [Q&A] [answered]
Is there any reason `useActionGroup` isn't exported from "react-aria" ?

I also want to know which package is the preferred one as I can see react-aria doesn't include some packages include in @react-aria/* packages.

**Comment by reidbarber** [comment, marked as answer]
The intent is to export things from `react-aria` once they get a stable release. It looks like that might be an oversight on our part, but we can get that fixed in the release. The `@react-aria/*` package is intended to be used while it is in pre-release, or if you only want to use an individual package (although using the monopackage is recommended.)

**Reply by ritz078** [reply]
Thank you. This answers my question. Do you want me to open an issue for exporting `useActionGroup` in react-aria?

**Reply by reidbarber** [reply]
I went ahead and made a PR [here](https://github.com/adobe/react-spectrum/pull/4432) since it was a quick fix. Thanks for finding!
